README.md

```diff
@@ -340,27 +344,27 @@ This is because by default, TensorBoard serves on host `0.0.0.0` which is
 publicly accessible. You can stop the popups by specifying `--host=localhost` at
 startup.
 
-### How can I develop TensorBoard?
+### How can I contribute to TensorBoard development?
 
-See [tensorflow/tensorboard/DEVELOPMENT.md](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorboard/DEVELOPMENT.md).
+See [DEVELOPMENT.md](https://github.com/tensorflow/tensorboard/blob/master/DEVELOPMENT.md).
 
 ### I have a different issue that wasn't addressed here!
 
 First, try searching our [GitHub
-issues](https://github.com/tensorflow/tensorflow/issues) and [Stack
+issues](https://github.com/tensorflow/tensorboard/issues) and [Stack
```

Comment:
Your paragraphs would flow nicer if you had a section at the bottom of this file for long URLs. For example:

```markdown
TensorBoard uses [Bazel] ≥0.5.1 as its build system, with
[installation instructions][bazel-install] here.

[Bazel]: http://bazel.io/
[bazel-install]: http://bazel.io/docs/install.html
```